libraries/AP_Mount/AP_Mount_Backend.h

```diff
@@ -83,6 +83,9 @@ class AP_Mount_Backend
     // If false (aka "follow") the gimbal's yaw is maintained in body-frame meaning it will rotate with the vehicle
```

Comment:
Sorry, it's an existing issue but we should add a comment above "set_yaw_lock" to clarify that it only applies in RC_TARGETING mode.  We should add a similar comment besides the _yaw_lock member declaration.